### ISSUE TITLE
[MPLUGIN-528] Tone done report plugin console output

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-plugin-tools#844]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** opened **[MPLUGIN-528](https://issues.apache.org/jira/browse/MPLUGIN-528?redirect=false)** and commented

The report plugin writes too much at `INFO` level:

```
[WARNING] ^
[INFO] Generating "Plugin Documentation" report --- maven-plugin-report-plugin:3.12.0:report
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/maven-metadata.xml
Downloading from apache.snapshots: https://repository.apache.org/snapshots/org/apache/maven/plugins/maven-pmd-plugin/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/maven-metadata.xml (1.4 kB at 20 kB/s)
Downloaded from apache.snapshots: https://repository.apache.org/snapshots/org/apache/maven/plugins/maven-pmd-plugin/maven-metadata.xml (862 B at 3.4 kB/s)
[INFO] Detecting plugin requirements history for range [0,): 36 releases, from 2.0-alpha-2 to 3.22.0
[INFO]   - RequirementsHistory{version='3.22.0', maven='3.2.5', jdk='8'}
[INFO]   - RequirementsHistory{version='3.21.2', maven='3.2.5', jdk='8'}
[INFO]   - RequirementsHistory{version='3.21.0', maven='3.2.5', jdk='8'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.20.0/maven-pmd-plugin-3.20.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.20.0/maven-pmd-plugin-3.20.0.pom (15 kB at 363 kB/s)
[INFO]   - RequirementsHistory{version='3.20.0', maven='3.2.5', jdk='8'}
[INFO]   - RequirementsHistory{version='3.19.0', maven='3.2.5', jdk='8'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.18.0/maven-pmd-plugin-3.18.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.18.0/maven-pmd-plugin-3.18.0.pom (15 kB at 392 kB/s)
[INFO]   - RequirementsHistory{version='3.18.0', maven='3.2.5', jdk='8'}
[INFO]   - RequirementsHistory{version='3.17.0', maven='3.2.5', jdk='8'}
[INFO]   - RequirementsHistory{version='3.16.0', maven='3.1.1', jdk='8'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.15.0/maven-pmd-plugin-3.15.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.15.0/maven-pmd-plugin-3.15.0.pom (14 kB at 313 kB/s)
[INFO]   - RequirementsHistory{version='3.15.0', maven='3.1.0', jdk='8'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.14.0/maven-pmd-plugin-3.14.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.14.0/maven-pmd-plugin-3.14.0.pom (13 kB at 371 kB/s)
[INFO]   - RequirementsHistory{version='3.14.0', maven='3.0', jdk='7'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.13.0/maven-pmd-plugin-3.13.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.13.0/maven-pmd-plugin-3.13.0.pom (12 kB at 213 kB/s)
[INFO]   - RequirementsHistory{version='3.13.0', maven='3.0', jdk='7'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.12.0/maven-pmd-plugin-3.12.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.12.0/maven-pmd-plugin-3.12.0.pom (12 kB at 374 kB/s)
[INFO]   - RequirementsHistory{version='3.12.0', maven='3.0', jdk='7'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.11.0/maven-pmd-plugin-3.11.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.11.0/maven-pmd-plugin-3.11.0.pom (13 kB at 341 kB/s)
[INFO]   - RequirementsHistory{version='3.11.0', maven='3.0', jdk='7'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.10.0/maven-pmd-plugin-3.10.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.10.0/maven-pmd-plugin-3.10.0.pom (12 kB at 338 kB/s)
[INFO]   - RequirementsHistory{version='3.10.0', maven='3.0', jdk='7'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.9.0/maven-pmd-plugin-3.9.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.9.0/maven-pmd-plugin-3.9.0.pom (12 kB at 356 kB/s)
[INFO]   - RequirementsHistory{version='3.9.0', maven='3.0', jdk='7'}
[INFO]   - RequirementsHistory{version='3.8', maven='2.2.1', jdk='7'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.7/maven-pmd-plugin-3.7.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.7/maven-pmd-plugin-3.7.pom (11 kB at 332 kB/s)
[INFO]   - RequirementsHistory{version='3.7', maven='2.2.1', jdk='7'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.6/maven-pmd-plugin-3.6.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.6/maven-pmd-plugin-3.6.pom (12 kB at 343 kB/s)
[INFO]   - RequirementsHistory{version='3.6', maven='2.2.1', jdk='6'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.5/maven-pmd-plugin-3.5.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.5/maven-pmd-plugin-3.5.pom (12 kB at 218 kB/s)
[INFO]   - RequirementsHistory{version='3.5', maven='2.2.1', jdk='6'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.4/maven-pmd-plugin-3.4.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.4/maven-pmd-plugin-3.4.pom (12 kB at 376 kB/s)
[INFO]   - RequirementsHistory{version='3.4', maven='2.2.1', jdk='6'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.3/maven-pmd-plugin-3.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.3/maven-pmd-plugin-3.3.pom (12 kB at 304 kB/s)
[INFO]   - RequirementsHistory{version='3.3', maven='2.2.1', jdk='6'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.2/maven-pmd-plugin-3.2.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.2/maven-pmd-plugin-3.2.pom (11 kB at 94 kB/s)
[INFO]   - RequirementsHistory{version='3.2', maven='2.2.1', jdk='6'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.1/maven-pmd-plugin-3.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.1/maven-pmd-plugin-3.1.pom (12 kB at 297 kB/s)
[INFO]   - RequirementsHistory{version='3.1', maven='2.2.1', jdk='5'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.0.1/maven-pmd-plugin-3.0.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.0.1/maven-pmd-plugin-3.0.1.pom (9.9 kB at 319 kB/s)
[INFO]   - RequirementsHistory{version='3.0.1', maven='2.2.1', jdk='5'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.0/maven-pmd-plugin-3.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/3.0/maven-pmd-plugin-3.0.pom (9.8 kB at 245 kB/s)
[INFO]   - RequirementsHistory{version='3.0', maven='2.2.1', jdk='5'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.7.1/maven-pmd-plugin-2.7.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.7.1/maven-pmd-plugin-2.7.1.pom (7.4 kB at 180 kB/s)
[INFO]   - RequirementsHistory{version='2.7.1', maven='2.2.1', jdk='5'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.7/maven-pmd-plugin-2.7.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.7/maven-pmd-plugin-2.7.pom (7.4 kB at 246 kB/s)
[INFO]   - RequirementsHistory{version='2.7', maven='2.2.1', jdk='5'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.6/maven-pmd-plugin-2.6.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.6/maven-pmd-plugin-2.6.pom (7.7 kB at 129 kB/s)
[INFO]   - RequirementsHistory{version='2.6', maven='2.0.6', jdk='5'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.5/maven-pmd-plugin-2.5.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.5/maven-pmd-plugin-2.5.pom (7.5 kB at 250 kB/s)
[INFO]   - RequirementsHistory{version='2.5', maven='2.0.6', jdk='1.4'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.4/maven-pmd-plugin-2.4.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.4/maven-pmd-plugin-2.4.pom (7.2 kB at 175 kB/s)
[INFO]   - RequirementsHistory{version='2.4', maven='2.0.6', jdk='1.4'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.3/maven-pmd-plugin-2.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.3/maven-pmd-plugin-2.3.pom (6.9 kB at 172 kB/s)
[INFO]   - RequirementsHistory{version='2.3', maven='2.0.6', jdk='1.4'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.2/maven-pmd-plugin-2.2.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.2/maven-pmd-plugin-2.2.pom (6.7 kB at 135 kB/s)
[INFO]   - RequirementsHistory{version='2.2', maven='2.0', jdk='null'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.1/maven-pmd-plugin-2.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.1/maven-pmd-plugin-2.1.pom (2.3 kB at 70 kB/s)
[INFO]   - RequirementsHistory{version='2.1', maven='2.0', jdk='null'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.0/maven-pmd-plugin-2.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.0/maven-pmd-plugin-2.0.pom (2.0 kB at 16 kB/s)
[INFO]   - RequirementsHistory{version='2.0', maven='null', jdk='null'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.0-beta-1/maven-pmd-plugin-2.0-beta-1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.0-beta-1/maven-pmd-plugin-2.0-beta-1.pom (1.1 kB at 9.2 kB/s)
[INFO]   - RequirementsHistory{version='2.0-beta-1', maven='null', jdk='null'}
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.0-alpha-2/maven-pmd-plugin-2.0-alpha-2.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-pmd-plugin/2.0-alpha-2/maven-pmd-plugin-2.0-alpha-2.pom (4.3 kB at 18 kB/s)
[INFO]   - RequirementsHistory{version='2.0-alpha-2', maven='null', jdk='null'}
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  02:13 min
[INFO] Finished at: 2024-06-08T00:16:24+02:00
[INFO] ------------------------------------------------------------------------
D:\Entwicklung\Projekte\maven-pmd-plugin [master ≡]>
```

That detail is only interesting at `DEBUG` level.

---

**Affects:** 3.12.0, 3.13.1

**Backported to:** [4.0.0-beta-2](https://github.com/apache/maven-plugin-tools/milestone/4?closed=1), [3.15.0](https://github.com/apache/maven-plugin-tools/milestone/43?closed=1)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

[Tamas Cservenak](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=cstamas), [Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [3e99eb0c0ea69bcc9220473ee3d28c0bcc4325d0](https://gitbox.apache.org/repos/asf?p=maven-plugin-tools.git;a=commit;h=3e99eb0c0ea69bcc9220473ee3d28c0bcc4325d0) and with [15206546e79d82a146fe009ac8c12831a0ed78c8](https://gitbox.apache.org/repos/asf?p=maven-plugin-tools.git;a=commit;h=15206546e79d82a146fe009ac8c12831a0ed78c8) for 3.x.
